src/components/Footer/Footer.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import logo from '../../footerlogo.svg';
+import {
+  Wrapper, Ul, Img, Li,
+} from './Footer.style';
```

Comment:
One hint: A nice way of importing styles is to write

```
import * as S from './Footer.style';
```

Then you can render the components with `<S.Wrapper>. I updated the code on `profy.dev` this way. There are some advantages which I list in the "Styles in a separate style file" section on [this page](https://profy.dev/project/reddit-timer/review-tips/where-to-put-styles) in case you're interested.

Reply:
awesome, I knew there had to be a simpler way to import these components lol. I'll give that section a look 👍